Gruntfile.js

```diff
@@ -212,6 +255,38 @@ module.exports = function(grunt) {
 
 	});
 
+	grunt.registerTask( 'kebabToCamelCase', 'Rename any svg attributes to camel case for react', function() {
+		var svgFiles = grunt.file.expand( { filter: 'isFile', cwd: 'svg-min/' }, [ '**/*.svg' ] );
+
+		// Add stuff
+		svgFiles.forEach( function( svgFile ) {
+
+			// Grab the relevant bits from the file contents
+			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
```

Comment:
Wondering about the `svg-min` folder. What's the point of it? And should we be camelCasing those props if we ever hope to use them anywhere outside of React? If they're not intended to be used outside React, does it need to be a separate folder?